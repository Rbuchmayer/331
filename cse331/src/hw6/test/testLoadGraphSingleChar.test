# load graphs with one character
LoadGraph oneCharOneBook oneCharOneBook.tsv
LoadGraph oneCharTwoBooks oneCharOneBook.tsv