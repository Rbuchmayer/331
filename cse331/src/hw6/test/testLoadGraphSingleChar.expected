# load graphs with one character
loaded graph oneCharOneBook
loaded graph oneCharTwoBooks